.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
it's typical to also have `--color` in here

Reply:
It's not in the default generated one FWIW, but I've added it.